src/seedu/addressbook/data/person/Birthday.java

```diff
@@ -0,0 +1,71 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import org.ocpsoft.prettytime.nlp.PrettyTimeParser;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Represents a Person's birthday in the address book.
+ * Guarantees: immutable;
+ */
+public class Birthday {
+    public final Date value;
+    private boolean isPrivate;
+
+    public static String EXAMPLE = "Sat Feb 18 16:03:28 SGT 2017";
```

Comment:
lol this is so that the test case can return the exact same Date (including time)